public/legacy/include/DetailView/DetailView2.php

```diff
@@ -123,6 +123,10 @@ public function setup(
             require("modules/$this->module/metadata/$metadataFileName.php");
         }
 
+        if (file_exists("custom/modules/$this->module/metadata/ext_metadata/$metadataFileName.php")) {
+            require("custom/modules/$this->module/metadata/ext_metadata/$metadataFileName.php");
+        }
```

Comment:
Hi @mpuyosa91, why are we adding this?

Reply:
Hi @clemente-raposo, the record-logic-config array-entry is something new for detailviewdefs; sometimes, the studio overwrites it, and a safe way to avoid this (See the example on the Test Environment) is to create an extra file that directly modifies it.